src/sentry_plugins/github/plugin.py

```diff
@@ -316,7 +316,7 @@ def create_repository(self, organization, data, actor=None):
                 data['name'], {
                     'name': 'web',
                     'active': True,
-                    'events': ['push'],
+                    'events': ['push', 'pull_request'],
```

Comment:
Not totally sure what the best way to handle existing repos is. There's an api for updating webhooks: https://developer.github.com/v3/repos/hooks/#edit-a-hook, but we need to know which user auth to use, and apart from having something in the Sentry UI that prompts users to "upgrade" to include PRs, I don't know how we'd figure out which user to use. The other thing we could do is just tell users with existing repos to update the events in the Github UI, but probably some people will do that incorrectly. This seems like something Matt might have an opinion on, so let's ask him.

Reply:
this is still an unresolved problem in the PR

Reply:
i wrote a script for this btw